Let's start with a simple mergebot integration.

orsenthil opened the pull request:
Hey @joerg84  - we need to settle upon the labels for the repo..

* This file is the config for mergebot integration.
* It will apply "requested-changed" as soon as PR is created. I think, we need a different label. What label should it be?

Commit:
Let's start with a simple mergebot integration.

joerg84 commented:
@orsenthil As discussed. How can I test this PR?

Commit:
Correct the label name

Correct the label name.

orsenthil commented:
@joerg84  - once we merge this PR, we give permissions to @mesosphere-mergebot-dev permissions to apply label and it will do something like this (This is test and proof: https://github.com/mesosphere-mergebot/mergebot-test-dcos/pull/144 ) For the config (https://github.com/mesosphere-mergebot/mergebot-test-dcos/blob/master/mergebot-config.json) that is same as the contents of this PR.

joerg84 commented on mergebot-config.json:
```diff
@@ -0,0 +1,7 @@
+  "label": {
+    "enabled": true,
+    "interactive": false,
+    "default": "requested-changes"
```
This shouldn't be `requested-changes` see here: https://github.com/joerg84/universe/pull/2

Commit:
Address review comments

Let's use `awaiting-testing-approval` as the PR label.

Commit:
Default Labels for PR and Intro

When a Mesosphere Employee Creates a PR, the label *"ready-for-review"* is applied by default. For an external contributor, "awaiting-testing-approval" label is applied whenever a PR is created.

Also, the given intro message is posted whenever an external contribution is received.
